src/build/yaml/ci-post-to-github-steps.yml

```diff
@@ -21,8 +21,42 @@ steps:
   displayName: 'Publish compat results to Artifacts'
   continueOnError: true
 
+- powershell: |
+    $OWNER = 'microsoft';
+    $REPO = 'Power-Fx';
+    $ISSUE_NUMBER = "$(System.PullRequest.PullRequestNumber)";
+    $stringToMatch = "*Binary Compatibility*";
+
+    $listCommentsUri = "https://api.github.com/repos/$OWNER/$REPO/issues/$ISSUE_NUMBER/comments";
+
+    $listResult = Invoke-RestMethod "$listCommentsUri";
+
+    $compatComments = ($listResult | Where-Object {$_.body -like $stringToMatch});
```

Comment:
Consider also filtering on $_.user.login == "BruceHaley", in case a PR Author replies to the compat comment and their reply also has "Binary Compatibility" in it, we probably don't want to delete that.